src/Admin_Tools_Command.php

```diff
@@ -0,0 +1,301 @@
+<?php
+
+/**
+ * Enables/Disables admin-tools on a site.
+ *
+ * ## EXAMPLES
+ *
+ *     # Enable admin tools on site
+ *     $ ee admin-tools up example.com
+ *
+ * @package ee-cli
+ */
+
+use EE\Model\Site;
+use \Symfony\Component\Filesystem\Filesystem;
+use \Composer\Console\Application;
+use \Symfony\Component\Console\Input\ArrayInput;
+
+class Admin_Tools_Command extends EE_Command {
+
+
+	/**
+	 * @var string $command Name of the command being run.
+	 */
+	private $command;
+
+	/**
+	 * @var Filesystem $fs Symfony Filesystem object.
+	 */
+	private $fs;
+
+	/**
+	 * @var array $db Object containing essential site related information.
+	 */
+	private $db;
+
+	public function __construct() {
+
+		$this->command = 'admin-tools';
+		$this->fs      = new Filesystem();
+		define( 'ADMIN_TOOL_DIR', EE_CONF_ROOT . '/admin-tools' );
+	}
+
+	/**
+	 * Installs admin-tools for EasyEngine.
+	 */
+	public function install() {
+
+		if ( ! $this->is_installed() ) {
+			EE::log( 'Installing admin-tools. This may take some time.' );
+			$this->fs->mkdir( ADMIN_TOOL_DIR );
+		}
+
+		$tools = json_decode( file_get_contents( ADMIN_TOOLS_FILE ), true );
+
+		foreach ( $tools as $tool => $data ) {
+			if ( ! $this->is_installed( $tool ) ) {
+				EE::log( "Installing $tool" );
+				$tool_path = ADMIN_TOOL_DIR . '/' . $tool;
+				call_user_func_array( [ $this, "install_$tool" ], [ $data, $tool_path ] );
```

Comment:
I think here we need to check method_exists since the $tool can come from json file which can be updated from `ADMIN_TOOLS_FILE` constant.